scripts/loader

```diff
@@ -100,4 +102,7 @@ if [ $sigterm_received -eq 0 ] && is_directory_link "${GG_ROOT}/alts/old" && is_
   flip_link "${GG_ROOT}/alts/old" "${LAUNCH_DIR}"
 fi
 
+## Touch an empty file to indicate rollback due to unexpected Nucleus exit
+touch "${GG_ROOT}/work/aws.greengrass.Nucleus/restart_panic"
```

Comment:
is it possible to just rely on the alts/broken dir link

Reply:
relying on the alts/broken dir links is not sufficient as any form of rollback will create these symlinks

Reply:
however, i could create a new symlink pointing to the absolute broken dir if letting loader touch a file is an issue